WordPressUtils/src/main/java/org/wordpress/android/util/PhotonUtils.java

```diff
@@ -138,6 +138,12 @@ public static String getPhotonImageUrl(String imageUrl, int width, int height, Q
             query += "&ssl=1";
         }
 
-        return "https://i0.wp.com/" + imageUrl.substring(schemePos + 3, imageUrl.length()) + query;
+        int beginIndex = schemePos + 3;
+        if (beginIndex < 0 || beginIndex > imageUrl.length()) {
+            // Fallback to original URL if the beginIndex is invalid to avoid `StringIndexOutOfBoundsException`
```

Comment:
NP: It seems that `imageUrl` is conditionally mutated a few times prior to this. Should we tweak this code comment to not mention "original" URL? Or, alternatively, if that is what we want, store it in a local var at the top of the function and return it here without mutation? 🤔

Reply:
That's a good suggestion @mkevins 👍 
In the other scheme error that the `imageUrl` is returned it hasn't been mutated yet. I think it makes sense to keep a copy of the url before any changes and return that.
I changed the code with e6b28eb2feacaf27fcb2196073be52688ce72e62